packages/experimental/Badge/src/PresenceBadge/PresenceBadgeTokens.ts

```diff
@@ -34,4 +35,20 @@ export const defaultPresenceBadgeTokens: TokenSettings<PresenceBadgeTokens> = ()
       width: 32,
       height: 32,
     },
+    available: getBadgeColor('lightGreen', t),
+    away: getBadgeColor('marigold', t),
+    awayOutOfOffice: getBadgeColor('berry', t),
+    busy: getBadgeColor('red', t),
+    blocked: getBadgeColor('red', t),
+    unknown: getBadgeColor('red', t),
+    offline: {
+      iconColor: t.host.appearance === 'highContrast' ? t.colors.neutralForeground3 : globalTokens.color.grey[38],
```

Comment:
Is this a case where we would eventually want to use 1 alias token (neutralForeground3), and then when the theme is HC, it maps that token to a different global token?

Reply:
badges in HC should have neutralForeground3. In all other themes they have exact colors which can be taken from global tokens

Reply:
There aren't alias tokens for the presence colors that I'm aware of, though we can add them in FURN specifically. I don;t think it's worth it though unless clients need the colors.